python/graphframes/examples.py

```diff
@@ -57,3 +60,216 @@ def friends(self):
         ], ["src", "dst", "relationship"])
         # Create a GraphFrame
         return GraphFrame(v, e)
+
+    def gridIsingModel(self, n, vStd=1.0, eStd=1.0):
+        """Grid Ising model with random parameters.
```

Comment:
this is quite a bit we are adding here, would it be better if this is a separate file like BeliefPropagation.scala is a separate file?